src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/data/CreateActivityRequest.kt

```diff
@@ -12,22 +10,18 @@ import java.time.LocalDate
 data class CreateActivityRequest(
 
   @Schema(description = "Code generated from the activity and schedule ids and mapped", required = true)
-  @NotBlank
```

Comment:
these were all annotated wrong so never worked and there were no tests for them, so removed all of them to avoid confusion